L81-core-cpp-jwt-token-audience.md

```diff
@@ -2,40 +2,47 @@ L81: Custom Audience in JWT Access Credentials and Google Default Credentials
 ----
 * Author(s): [Jiangtao Li](https://github.com/jiangtaoli2016), [Yihua Zhang](https://github.com/yihuazhang)
 * Approver: markdroth
-* Status: Implemented {Draft, In Review, Ready for Implementation, Implemented}
+* Status: Draft {Draft, In Review, Ready for Implementation, Implemented}
 * Implemented in: Core, C++
-* Last updated: 2021-06-04
+* Last updated: 2021-06-29
 * Discussion at: https://groups.google.com/g/grpc-io/c/MgSP5tNrM0Q
 
 ## Abstract
 
-In current gRPC C++ implementation, the audience field of JWT access token is
-always set to the service URL. It works for most of the cases. However, there
-are use cases where a user wants to override the default audience (service URL)
-and uses its own. This gRFC provides a way for an applications to override
-the default audience field.
+In current gRPC C++ implementation, it does not allow to include the scope
```

Comment:
Since this is now about setting scope instead of setting audience, please also change the gRFC title and rename the file.

Reply:
Done.